src/dpp/discordvoiceclient.cpp

```diff
@@ -1062,6 +1068,9 @@ size_t discord_voice_client::encode(uint8_t *input, size_t inDataSize, uint8_t *
 		throw dpp::voice_exception("Invalid input data length: " + std::to_string(inDataSize) + ", must be n times of " + std::to_string(mEncFrameBytes));
 	}
 #else
+	(void)input;
```

Comment:
don't these generate another warning about a statement that has no effect?

Reply:
Not that I'm seeing. I was wary of adding `[[maybe_unused]]` because the arguments are used if the voice_client is enabled.

Honestly, it might make more sense for the function to only exist if `HAVE_VOICE` is defined.